src/components/interface/SwapInfo/SwapInfo.tsx

```diff
@@ -92,11 +92,16 @@ export const SwapInfo: React.FunctionComponent<SwapInfoProps> = ({
       )}
 
       {(mode === SwapFormModes.NEW_POSITION || mode === SwapFormModes.FIX_BORROW) &&
-        !isUndefined(warningText) && (
+        !isUndefined(warningBoxes) && (
           <>
-            <Box sx={bottomSpacing}>
-              <WarningBox warningText={warningText} />
-            </Box>
+            {warningBoxes.map((warningBox) => {
+              return (
+                // eslint-disable-next-line react/jsx-key
```

Comment:
there is a reason why we need to use keys for Lists and why this is enforced via Lint. Please remove the eslint-disable-next-line react/jsx-key and pass a key to the Box component.

Documentation: https://reactjs.org/docs/lists-and-keys.html

Reply:
adressed